sourced/ml/algorithms/uast_ids_to_bag.py

```diff
@@ -70,3 +72,23 @@ def __init__(self, token2index=None, token_parser=None):
         """
         token_parser = TokenParser() if token_parser is None else token_parser
         super().__init__(token2index, token_parser)
+
```

Comment:
As Konst found, `roleQualified` should be removed

Reply:
yes, checked it in https://github.com/src-d/ml/pull/207 - will be removed